examples/tutorials/01_refactor_code.ipynb

```diff
@@ -13,11 +13,14 @@
    "id": "59fc3ea7",
    "metadata": {},
    "source": [
-    "LineaPy’s artifacts store the complete development history in intelligent graph representation, and this enables automatic transformation of the development code.\n",
+    "Data science development is characterized by nimble, iterative experimentation and exploratory data analysis.\n",
+    "Data scientists explore many possibilities before reaching the final result. The rapid exploration process often\n",
+    "leads to long, messy code, over 90% of which has no impact on the final result.\n",
```

Comment:
i would prefer staying away from hurtful figures like 'over 90%' in this context. Also unless we are ready to back this number, it goes out.

Reply:
@dorx